vm/starcoin-transactional-test-harness/src/lib.rs

```diff
@@ -1113,8 +1113,10 @@ impl<'a> MoveTestAdapter<'a> for StarcoinTestAdapter<'a> {
             } else {
                 None
             };
+
+            let network = init_args.network.unwrap_or(BuiltinNetworkID::Test);
```

Comment:
Better to change the networkID to test in move test scripts. or change DEV genesis config to v12 stdlib if it works.